src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -18,15 +18,15 @@ pub type RawExtensions<'a> = common::Asn1ReadableOrWritable<
 ///
 /// In particular, an `Extensions` cannot be constructed from a `RawExtensions`
 /// that contains duplicated extensions (by OID).
-pub struct Extensions<'a>(Option<RawExtensions<'a>>);
+pub struct Extensions<'a>(Option<&'a RawExtensions<'a>>);
```

Comment:
I added this inner `'a` bound, but it isn't strictly necessary. Happy to remove.